app/src/main/java/nu/yona/app/ui/dashboard/DayActivityDetailFragment.java

```diff
@@ -256,19 +240,47 @@ public void onPageSelected(int position)
 				@Override
 				public void onPageScrollStateChanged(int state)
 				{
-
 				}
 			});
 		}
+	}
 
-		YonaApplication.getEventChangeManager().registerListener(this);
-		setHook(new YonaAnalytics.BackHook(AnalyticsConstant.BACK_FROM_DAY_ACTIVITY_DETAIL_SCREEN));
-		return view;
+	private void previousDayActivity()
```

Comment:
It's unclear what this method does. It's name sounds like a statement instead of an action (or a condition, in case of a boolean-returning ``isXxx`` method). It should be something like ``enablePreviousDayActivity``, ``goToPreviousDayActivity``, ``showPreviousDayActivity``, etc.